src/lib/ui/keyboard-avoiding-view/component.tsx

```diff
@@ -0,0 +1,47 @@
+import { KeyboardAwareScrollView } from '@pietile-native-kit/keyboard-aware-scrollview';
+import { ReactElement, ReactNode } from 'react';
+import { KeyboardAvoidingView, Platform, ScrollViewProps, StyleSheet, ViewStyle } from 'react-native';
+import { useSafeAreaInsets } from 'react-native-safe-area-context';
+
+export interface AppKeyboardAvoidingViewProps extends Partial<ScrollViewProps> {
+  children: ReactNode;
+  withoutScrollView?: boolean;
```

Comment:
How about? To match `ScrollViewProps`:

```suggestion
  scrollEnabled?: boolean;
```

Reply:
As discussed - removed